File: Python/Program/Product.py
# Saya Jasmine Noor Fawzia [2200598] mengerjakan soal LP2 dalam Mata Kuliah DPBO
# untuk keberkahan-Nya maka saya tidak melakukan kecurangan seperti yang telah dispesifikasikan
# Aamiin

# deklarasi class
class Product:
    # constructor
    def __init__(self, idProduct, name, brand, price):
        self.idProduct = idProduct
        self.name = name
        self.brand = brand
        self.price = price
    
    # get id
    def get_id(self):
        return self.idProduct
    
    # set id
    def set_id(self, idProduct):
        self.idProduct = idProduct
    
    # get name
    def get_name(self):
        return self.name
    
    # set name
    def set_name(self, name):
        self.name = name
    
    # get brand
    def get_brand(self):
        return self.brand
    
    # set brand
    def set_brand(self, brand):
        self.brand = brand
    
    # get price
    def get_price(self):
        return self.price
    
    # set price
    def set_price(self, price):
        self.price = price

File: PHP/Program/Product.php
<!-- 
Saya Jasmine Noor Fawzia [2200598] mengerjakan soal LP1 dalam Mata Kuliah DPBO
untuk keberkahan-Nya maka saya tidak melakukan kecurangan seperti yang telah dispesifikasikan
Aamiin 
-->

<?php
//deklarasi class
class Product{
    public $idProduct;
    public $name;
    public $brand;
    public $price;
  
    //constructor
    function __construct($idProduct, $name, $brand, $price){
        $this->idProduct = $idProduct;
        $this->name = $name;
        $this->brand = $brand;
        $this->price = $price;
    }

    //set id
    public function setId($idProduct){
        $this->idProduct = $idProduct;
    }

    //get id
    public function getId(){
        return $this->idProduct;
    }

    //set name
    public function setName($name){
        $this->name = $name;
    }

    //get name
    public function getName(){
        return $this->name;
    }

    //set brand
    public function setBrand($brand){
        $this->brand = $brand;
    }

    //get brand
    public function getBrand(){
        return $this->brand;
    }

    //set price
    public function setPrice($price){
        $this->price = $price;
    }

    //get price
    public function getPrice(){
        return $this->price;
    }
}
?>